asreviewcontrib/datatools/dedup.py

```diff
@@ -102,4 +103,59 @@ def drop_duplicates_by_similarity(
     if verbose:
         _print_similar_list(similar_list, data)
 
-    asdata.df = asdata.df[~duplicated].reset_index(drop=True)
\ No newline at end of file
+    asdata.df = asdata.df[~pd.Series(duplicated)].reset_index(drop=True)
+
+
+def deduplicate_data(asdata: ASReviewData, args: Namespace) -> None:
+    initial_length = len(asdata.df)
+
+    if args.pid not in asdata.df.columns:
+        print(
+            f"Not using {args.pid} for deduplication "
+            "because there is no such data."
+        )
+
+    if not args.similar:
```

Comment:
Now you can't use both kinds of deduplication at the same time. Did you have a reason for that? I think it would be easier to use if you can just do `asreview data dedup file.csv --similar` and you get a fully deduplicated set, instead of having to run deduplication twice, once with the `--similar` flag and once without.

Reply:
If you run with the similar flag it will also catch the cases where it is an exact duplicate, since the similarity score will be higher than of a just similar case.

Reply:
But you will not catch duplicates based on PID, right?

Reply:
When checking for similar papers it won't, but that's kind of the point, isn't it?

Reply:
If I want to remove duplicates based on PID and on similar title/abstract, I have to call `asreview data dedup file.csv` and then call `asreview data dedup file.csv --similar`. I just thought it'd might be nice to call `asreview data dedup file.csv --similar` and get both kinds of deduplication in one go. But if you like it better if they are separate, I'm also fine with it. I don't have a strong opinion.

Reply:
It is not that I am against it, it is more like, if the pid is different it is high likely that the title won't be the same. But maybe this could find cases where the pid is the same but the titles completely different, and warn the user about it. I will include it in the solution as well.